app/models/work.rb

```diff
@@ -0,0 +1,34 @@
+class Work < ApplicationRecord
+  validates :title, presence: true, uniqueness: true
+  has_many :votes
+  has_many :users, through: :votes
+
+  def vote_counter
+    return self.votes.length
+  end
+
+  def self.spotlight_work(works)
```

Comment:
Hm... You designed this method so that it took in `works` as a parameter. In a different project, this would be because you wanted this method to be very flexible: what if we wanted to do a spotlight_work with only albums? Or with only recently created works? That's cool and forward-thinking! However, in this specific project, we can probably just assume that you will be looking at all works, so `Work.all`... I wouldn't have thought it as a requirement to pass in `works`